account-kit/signer/src/client/index.ts

```diff
@@ -205,19 +229,21 @@ export class AlchemySignerWebClient extends BaseSignerClient<ExportWalletParams>
    *  },
    * });
    *
-   * const account = await client.completeEmailAuth({ orgId: "user-org-id", bundle: "bundle-from-email" });
+   * const account = await client.completeAuth({ orgId: "user-org-id", bundle: "bundle-from-email", connectedEventName: "connectedEmail" });
    * ```
    *
    * @param {{ bundle: string; orgId: string }} config The configuration object for the authentication function containing the credential bundle to inject and the organization id associated with the user
    * @returns {Promise<User>} A promise that resolves to the authenticated user information
    */
-  public completeEmailAuth = async ({
+  public override completeAuth = async ({
     bundle,
     orgId,
+    connectedEventName,
   }: {
     bundle: string;
     orgId: string;
-  }) => {
+    connectedEventName: keyof AlchemySignerClientEvents;
+  }): Promise<User> => {
     this.eventEmitter.emit("authenticating");
     await this.initIframeStamper();
```

Comment:
maybe we should rename this to `completeBundleAuth` or something since passkey auth doesn't use the iframe stamper? I have a branch somewhere that tries to add passkey sessions, but I hit a roadblock in the signer-service which needed a change to the chain configs so the branch is private.

This method is great for that branch so thanks for doing this refactor!